### DISCUSSION TITLE
I can't into my  space settings

### DISCUSSION THREAD

**Discussion by mtf0919** [snapshot-labs/snapshot#1112] [Q&A]
this is picture ，not virus。please help me！
http://a1.qpic.cn/psc?/V14FWZAO1EUMRZ/ibw3PBAeGGFcX*BnzIQux0xBTo3VdlVv1EJaf4pou*OliZIvPjrLKdsXY5qZYYkr3csTPFDNCgZGvOUD19qySg!!/b&ek=1&kp=1&pt=0&bo=WQWAAkcHZwMDAN8!&tl=1&vuin=99909181&tm=1637744400&sce=50-1-1&rf=4-0

http://a1.qpic.cn/psc?/V14FWZAO1EUMRZ/ibw3PBAeGGFcX*BnzIQux99n5248eiE*fxGNovQ.LTNvdidI6BktZInedMK3jkFLc6W2HJKjiDT7ktdqH5820g!!/b&ek=1&kp=1&pt=0&bo=8wUxAvMFMQIDACU!&tl=1&vuin=99909181&tm=1637744400&sce=50-1-1&rf=4-0
every step is OK！

why i cannt into my space settings？？

please help me！thx！！！

**Comment by metrox-eth** [comment]
Q&A is a community curated FAQ so if you have a support question it's preferable to ask on our Discord #helpdesk channel.
Is your problem solved now ? If yes, I will delete this question in a few days.
